scripts/check-pack.mjs

```
@@ -0,0 +1,73 @@
+// and `types`), instead of a hand-written list of paths. The previous check named
+// `dist/index.js` and `dist/index.d.ts` literally, so `./view` — which MulmoTerminal's
+// headless preview and MulmoServer's `AppViewFrame.vue` both import — could vanish from the
+// tarball and still pass. A list that is written out by hand reopens that hole the day a
+// third subpath is added; a list that is derived cannot.
+//
+// Usage: node scripts/check-pack.mjs <tarball>
+
+import { execFileSync } from "node:child_process";
+import { readFileSync } from "node:fs";
+
+const tarball = process.argv[2];
```

ESLint

[error] 'process' is not defined.

(no-undef)